src/service/mercury/helpers/transformers.ts

```diff
@@ -24,13 +40,46 @@ const transformAccountBalances = async (
   rawResponse: OperationResult<MercuryAccountBalancesData>,
   tokenDetails: TokenDetails
 ) => {
-  return rawResponse?.data?.entryUpdateByContractIdAndKey.nodes.map((entry) => {
+  const data = rawResponse?.data?.entryUpdateByContractIdAndKey.nodes || [];
+  const formattedBalances = data.map((entry) => {
     const details = tokenDetails[entry.contractId];
+    const totalScVal = xdr.ScVal.fromXDR(Buffer.from(entry.valueXdr, "base64"));
     return {
       ...entry,
       ...details,
+      total: scValToNative(totalScVal),
     };
   });
+
+  const balances = formattedBalances.reduce((prev, curr) => {
+    if (curr.symbol === "XLM") {
+      prev["native"] = {
+        token: { type: "native", code: "XLM" },
+        total: new BigNumber(curr.total),
+        available: new BigNumber(curr.total), // TODO: how to get available for xlm?
```

Comment:
I think this is total - (base reserver * ops)? Not sure if this applies outside of the context of a transaction?

Reply:
I think this equivalent `minimum balance` right? The formula is a bit more complicated and can be found here: https://developers.stellar.org/docs/fundamentals-and-concepts/lumens#minimum-balance